Makefile

```diff
@@ -46,20 +46,35 @@ $(OBJDIR)/%.o: %.cpp
 $(OBJDIR)/%.o: %.ispc
 	$(ISPC) $(ISPCFLAGS) -o $@ $^
 
+%.o: %.ispc
+	$(ISPC) $(ISPCFLAGS) -o $@ $^
+
 $(OBJDIR):
 	mkdir -p $@
 
+$(OBJDIR)/stream_tasks_loopy.ispc: loopy-install $(OBJDIR)
+	loopy-install/bin/python3 gen-loopy.py > $@
 
 $(OBJDIR)/stream:      CFLAGS+=-mcmodel=medium $(STREAM_DEFINES)
 $(OBJDIR)/stream_ispc: CFLAGS+=$(STREAM_DEFINES)
 $(OBJDIR)/stream_ispc: ISPCFLAGS+=$(STREAM_DEFINES)
 $(OBJDIR)/stream_ispc: LDLIBS+=-lstdc++
 
+$(OBJDIR)/stream_ispc_loopy: CFLAGS+=$(STREAM_DEFINES)
+$(OBJDIR)/stream_ispc_loopy: ISPCFLAGS+=$(STREAM_DEFINES)
+$(OBJDIR)/stream_ispc_loopy: LDLIBS+=-lstdc++
+
 # Dependencies
 $(OBJDIR)/stream: $(OBJDIR)/stream.o | $(OBJDIR)
 $(OBJDIR)/stream.o: stream.c | $(OBJDIR)
 
 $(OBJDIR)/stream_ispc: $(OBJDIR)/stream_ispc.o $(OBJDIR)/stream_tasks.o $(OBJDIR)/tasksys.o | $(OBJDIR)
 $(OBJDIR)/stream_ispc.o: stream_ispc.c | $(OBJDIR)
+$(OBJDIR)/stream_ispc_loopy: $(OBJDIR)/stream_ispc.o $(OBJDIR)/stream_tasks_loopy.o $(OBJDIR)/tasksys.o | $(OBJDIR)
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^ $(LDLIBS)
```

Comment:
I couldn't figure out how to avoid stating this. I guess I haven't really worked with Make in a while. Help? :slightly_smiling_face:

Reply:
I was able to remove line 74 without issue.

Reply:
Weird. Without that line (i.e. with the version of things you pushed), it refuses to build `stream_ispc_loopy` for me.

```
GNU Make 4.4.1
Built for x86_64-pc-linux-gnu
Copyright (C) 1988-2023 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

Reply:
And `make -d` says
```
  Successfully remade target file 'build/stream_tasks_loopy.o'.
  Pruning file 'build/tasksys.o'.
  Pruning file 'build'.
 Finished prerequisites of target file 'build/stream_ispc_loopy'.
 Must remake target 'build/stream_ispc_loopy'.
 Successfully remade target file 'build/stream_ispc_loopy'.
```
I.e. it seems to think no commands need to be run to make `stream_ispc_loopy`.

Reply:
Hmm, I see that now as well.  I swear it work before.  Let me look into it.

Reply:
Boy I have no idea what is going on.

Reply:
I see now.  Our rule was not matching because the first object in the dependencies didn't match the binary name.  So, I implemented a hack https://github.com/lcw/stream_ispc/commit/e56dd2d15e96ecc768ea01dc827d21dc3be1aeab.

Reply:
OIC, that makes sense. Thanks for figuring it out!